airbyte-webapp/src/locales/en.json

```diff
@@ -390,6 +390,8 @@
   "tables.name": "Name",
   "tables.connector": "Connector",
   "tables.connections": "Connections",
+  "tables.connections.changes.breaking": "Platform and Connector don’t use the same protocol version",
+  "tables.connections.changes.non_breaking": "We detected some non-breaking change on the source",
```

Comment:
I think we should work on the wording here a bit. Those errors are currently very "non helpful" in a way that they are very technical and don't tell the user what's actually asked from them to do here. We should try to phrase them in a way that makes clear what this warnings/errors are meaning for their data as well as what we expect them to do to solve it.

@andyjih can maybe help with this.

Reply:
We can reuse the wording in #19226 for these. Also, breaking does not mean that the protocol has changed. That is a different feature that has not been implemented in the frontend yet.

Reply:
+1 to @edmundito on using the same wording there